Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -17,6 +17,8 @@ public protocol WebViewNavigationDelegate: AnyObject {
         shouldLoad request: URLRequest,
         navigationAction: WKNavigationAction
     ) async -> Bool
+    
+    func showWebViewError()
```

Comment:
It nice and can work with passing the error, but how about having a more detailed method like 
```
func webView(
        _ webView: WKWebView,
        didFail navigation: WKNavigation,
        withError error: Error
    )
 ```   
Thoughts?

Reply:
In our use case, displaying specific error details is not required. We are currently only showing internet error or generic error. Therefore, detailed information is not required at this time. We can reconsider this approach if there arises a need for more detailed error messages in the future.